### DISCUSSION TITLE
Scanning for k8s vulnerabilities

### DISCUSSION THREAD

**Discussion by jon-rei** [aquasecurity/trivy-operator#1619] [Help & Support] [answered]
Since trivy now supports scanning for k8s vulns added with https://github.com/aquasecurity/trivy/pull/5268 and https://github.com/aquasecurity/trivy/pull/5418. I was wondering if it's already possible to use the operator to do these scans.

**Comment by chen-keinan** [comment, marked as answer]
@jon-rei  not yet , but its on the plan and will be added soon I have created issue for it #1621
